src/layout/component/code-demo.js

```diff
@@ -41,7 +43,7 @@ const CodeDemo = ({ iframe, iframeFullwidth, height = null, code, _body, _ID, _r
 								<div className="code-demo__example__code">
 									<div className="tabs">
 										<nav className="tabs-nav">
-											<ul className="au-link-list au-link-list--inline ">
+											<ul className="au-link-list au-link-list--inline" role="tablist">
 												{
```

Comment:
is the `role=tab` now supposed to be on the `<li>`

Reply:
ignore that^ 
I think the `<li>` should now how `role="presentation"` though right?

Reply:
No, so to use `role="tab"` it needs to be in a container with an aria-role. The recommendation was `tablist`.